src/libraries/Native/Unix/System.Globalization.Native/pal_idna.c

```diff
@@ -15,13 +15,17 @@ enum {
 };
 #endif
 
+enum {
+    INCLUDE_ASCII_FLAGS=0x1,
+    INCLUDE_UNICODE_FLAGS=0x2
+};
+
 static const uint32_t AllowUnassigned = 0x1;
 static const uint32_t UseStd3AsciiRules = 0x2;
 
-static uint32_t GetOptions(uint32_t flags)
+static uint32_t GetOptions(uint32_t flags, uint32_t defaultFlags)
```

Comment:
>defaultFlags [](start = 52, length = 12)

this can be simple bool flag. no need to define enum for that

Reply:
Started with that idea, I did it the enum way to make it clearer from the call site perspective, but I can change it to bool.